test/integration/gardenlet/seed/seed/seed_test.go

```diff
@@ -333,12 +369,18 @@ var _ = Describe("Seed controller tests", func() {
 					By("Delete Seed")
 					Expect(testClient.Delete(ctx, seed)).To(Succeed())
 
-					By("Verify that the seed system components have been deleted")
-					Eventually(func(g Gomega) []resourcesv1alpha1.ManagedResource {
-						managedResourceList := &resourcesv1alpha1.ManagedResourceList{}
-						g.Expect(testClient.List(ctx, managedResourceList, client.InNamespace(testNamespace.Name))).To(Succeed())
-						return managedResourceList.Items
-					}).Should(BeEmpty())
+					if seedIsGarden {
+						// The CRDs are cleaned up by the Destroy function of GRM, in case the seed is not garden. So It might happen that, before we fetch the
```

Comment:
I find this comment confusing to talk about "the seed is not garden" while L372 has `if seedIsGarden`. Can you re-arrange this a bit to make it easier understandable?